src/components/Main/Header.tsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { memo } from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todosList: Todo[],
+  newTodoField: React.RefObject<HTMLInputElement>,
+  setIsError: (value: boolean) => void,
+};
+
+export const Header: React.FC<Props> = memo(({
+  todosList,
+  newTodoField,
+  setIsError,
+}) => {
+  return (
+    <header className="todoapp__header">
+      {todosList.length > 0 && (
```

Comment:
```suggestion
      {todosList.length && (
```

Reply:
otherwise I see 0 on page refresh😬

Reply:
> otherwise I see 0 on page refresh😬

Yeah, my bad, you can use !!todosList.length or leave your condition.